src/components/Modal.js

```diff
@@ -62,14 +63,17 @@ export default function Modal({
   setFilters,
   onButtonClick,
   onBadgeClick,
-  handleChange
+  selectedValue
 }) {
   const badgeKeys = Object.keys(filters);
 
   return (
     <>
       <Container show={show}>
-        <Title>Choose Contract Duration</Title>
+        <Title>
+          Choose Contract Duration <br />
+          (Months)
+        </Title>
         {/* <DurationBar /> */}
         <BadgeContainer>
           <BadgeDuration
```

Comment:
you could use `.map()` on `badgeKeys` to increase efficiency here, same for the next `<BadgeContainer>`